mappings/net/minecraft/util/Util.mapping

```diff
@@ -41,9 +41,11 @@ CLASS net/minecraft/class_156 net/minecraft/util/SystemUtil
 	METHOD method_648 getMeasuringTimeNano ()J
 	METHOD method_650 getValueAsString (Lnet/minecraft/class_2769;Ljava/lang/Object;)Ljava/lang/String;
 	METHOD method_651 getJVMFlags ()Ljava/util/stream/Stream;
-	METHOD method_652 thenCombine (Ljava/util/List;)Ljava/util/concurrent/CompletableFuture;
+	METHOD method_652 combine (Ljava/util/List;)Ljava/util/concurrent/CompletableFuture;
 		ARG 0 futures
-	METHOD method_654 consume (Ljava/lang/Object;Ljava/util/function/Consumer;)Ljava/lang/Object;
+	METHOD method_654 initialize (Ljava/lang/Object;Ljava/util/function/Consumer;)Ljava/lang/Object;
```

Comment:
If you rename this one to initialize, I think the supplier one below can be renamed as well.

Reply:
It would be weird calling the other one `initialize` since it just creates something using a supplier, while this one initializes a given object by applying a consumer to it.

I do agree that they should have the same name since they're used for the same purpose, so I'll name them both `create`.

Reply:
Both function as `setup` or so. sure